### DISCUSSION TITLE
How would I use a discord webhook on a project I'm not part of to see commits?

### DISCUSSION THREAD

**Discussion by gunvalk** [community/community#23826] [New to GitHub]
<p>I’d like to be able to use the webhook features to make a discord channel that shows the commit updates, but I cant figure out how you do that if you arent part of the project. Would you fork it and do it that way, or would that not automatically stay up to date?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/gunvalk">@gunvalk</a>,</p>
<p>Thanks for being here! I recommend reading this guide:</p>
<p><a href="https://support.discordapp.com/hc/en-us/articles/228383668-Intro-to-Webhooks" rel="nofollow noopener">https://support.discordapp.com/hc/en-us/articles/228383668-Intro-to-Webhooks</a></p>
<p>and this guide for GitHub’s webhooks in general:</p>
<p><a href="https://developer.github.com/webhooks/creating/" rel="nofollow noopener">https://developer.github.com/webhooks/creating/</a></p>
<p>If you have any questions about Discord and how to integrate with Discord from GitHub – I recommend contacting Discord support: <a href="https://support.discord.com/hc/en-us" rel="nofollow noopener">https://support.discord.com/hc/en-us</a></p>

**Comment by gunvalk** [comment, marked as answer]
<p>I got my question answered by someone on the server. If anyone else is wondering about a way to post commits to discord without having access to the webhooks, then you can use Discord rss bot and link it to the atom page for the repo.</p>
